students/ravi/Session02/fizz_buzz.py

```diff
@@ -0,0 +1,11 @@
+for i in range(100):
+    if (i+1)%3+(i+1)%5==0:
+        print('FizzBuzz')
+    else:
```

Comment:
You can use an "if" statement here to avoid all the nested if statements below. If the first if statement doesn't execute, it will move on to the next line of code.